Week-23/todo-ui/src/App.jsx

```diff
@@ -0,0 +1,15 @@
+import { useState } from 'react'
+import './App.css'
+import AddTodo from './pages/AddTodo'
+
+function App() {
+  const [count, setCount] = useState(0)
+
+  return (
+    <div>
+      <AddTodo/>
```

Comment:
Refactor your component to just Todo instead of AddTodo, because it does all the CRUD.